ocrd_calamari/recognize.py

```diff
@@ -31,6 +31,14 @@ def _init_calamari(self):
         checkpoints = glob(self.parameter['checkpoint'])
         self.predictor = MultiPredictor(checkpoints=checkpoints)
 
+        self.input_channels = self.predictor.predictors[0].network.input_channels
```

Comment:
Maybe `self.network_input_channels` would be less ambiguous? I would say `self.input_channels` is misleading as it does not pertain to the input but the used model aka checkpoint.

Reply:
Yes, good point.